java/code/src/com/redhat/rhn/manager/kickstart/cobbler/CobblerSystemCreateCommand.java

```diff
@@ -322,6 +329,21 @@ public ValidatorError store(boolean saveCobblerId) {
             }
         }
 
+        if (recProfile != null && "suse".equals(recProfile.getDistro().getBreed())) {
+            if (kernelOptions != null && kickstartHost != null && mediaPath != null &&
+                    !kernelOptions.contains("self_update=") && ksData != null) {
+                Optional<Channel> installerUpdated = ksData.getTree().getChannel()
+                        .getAccessibleChildrenFor(user)
+                        .stream()
+                        .filter(Channel::isInstallerUpdates)
+                        .findFirst();
+                if (installerUpdated.isPresent()) {
+                    kernelOptions = kernelOptions + " self_update=http://" + kickstartHost + "/ks/dist/child/" +
+                            installerUpdated.get().getLabel() + "/" + ksData.getTree().getLabel();
+                }
```

Comment:
I think you should "disable" it in the "else case". See `TreeCreateOperation.java` . You can do it with `self_update=0`

Reply:
right, thanks for noticing it

Reply:
not the last one. This is when self_update exists in kernelOptions. I think we keep it in this case